src/core/music_score_bar_labels.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "music_score_events.h"
+#include <string>
+/**
+ * Represents a label for a bar of the score.
+ */
+struct MusicScoreBarLabel {
+
+	/**
```

Comment:
Could this be changed to:

using MusicScoreBarLabel = std::string;

I'm not sure if we need to create an entire class when a typedef (or using) would suffice?